ebpfsvc/api.cpp

```diff
@@ -3,9 +3,17 @@
  *  SPDX-License-Identifier: MIT
  */
 
-#include "pch.h"
+#include "svc_common.h"
 #include <stdio.h>
 #include "rpc_interface_h.h"
+#include "ebpf_windows.h"
+#include "Verifier.h"
+#include "api_internal.h"
```

Comment:
can these be listed in the following order?
1) system includes using <>, in alphabetical order
2) others using "", in alphabetical order

```suggestion
#include <stdio.h>
#include "api_internal.h"
#include "ebpf_windows.h"
#include "rpc_interface_h.h"
#include "svc_common.h"
#include "Verifier.h"
```
This makes it easier to verify things like no duplicate includes